docs/guides/writing_clusters.md

```diff
@@ -211,9 +211,15 @@ Your cluster implementation must provide public getter and setter APIs for each
 attribute to allow applications to interact with cluster state.
 
 -   **Getter Methods:** Provide a getter method for every attribute (e.g.,
-    `GetCurrentSensitivityLevel()`, `GetAlarmsActive()`).
+    `GetCurrentSensitivityLevel()`, `GetAlarmsActive()`). Applications need
+    these to read the current cluster state.
+
 -   **Setter Methods:** Provide setter methods for all non-fixed (mutable)
-    attributes (e.g., `SetAlarmsActive()`, `SetCurrentSensitivityLevel()`).
+    attributes (e.g., `SetAlarmsActive()`, `SetCurrentSensitivityLevel()`) to
+    allow applications to make spec-compliant modifications to the cluster
```

Comment:
The point is, that "spec compliant" may involve something that sets multiple attributes together, not a single setter per attribute.  We should make this very clear.